doc/make.bat

```diff
@@ -1,112 +0,0 @@
-@ECHO OFF
```

Comment:
Is this not something auto-provided by Sphinx? I realize that nipype doesn't work well under Windows, but do we want to eliminate this?

Reply:
Not for windows, AFAIK. I would be happy to re-add if someone badly needs it. Otherwise, it seems it would be more than sufficient that RTD builds up the docs correctly.

Reply:
#3127